addons/main/script_macros_common.hpp

```diff
@@ -15,11 +15,11 @@
    - Provide a solid structure that can be dynamic and easy editable (Which sometimes means we cannot adhere to Aim #1 ;-)
      An example is the path that is built from defines. Some available in this file, others in mods and addons.
 
- Follows  Standard:
-   Object variables: PREFIX_COMPONENT
+ Follows Standard:
+   Object variables: PREFIX_COMPONENT (or PREFIX_COMPONENT_SUBCOMPONENT)
    Main-object variables: PREFIX_main
-   Paths: MAINPREFIX\PREFIX\SUBPREFIX\COMPONENT\SCRIPTNAME.sqf
-   e.g: x\six\addons\sys_menu\fDate.sqf
+   Paths: MAINPREFIX\PREFIX\SUBPREFIX\COMPONENT\SCRIPTNAME.sqf (or MAINPREFIX\PREFIX\SUBPREFIX\COMPONENT\SUBCOMPONENT\SCRIPTNAME.sqf)
```

Comment:
paths aren't being updated in this PR
```
#define PATHTOF(var1) PATHTOF_SYS(PREFIX,COMPONENT,var1)
#define PREP(var1) ['PATHTO_SYS(PREFIX,COMPONENT_F,DOUBLES(fnc,var1))', 'TRIPLES(ADDON,fnc,var1)'] call SLX_XEH_COMPILE_NEW
```

I would just say something like "SUBCOMPONENT are only for configs"

Reply:
`PREP` is half-updated, reckong that's fine?

Maybe a better name would be `SUBCONFIG` instead of `SUBCOMPONENT` then?